src/StaticCaching/Cachers/FileCacher.php

```diff
@@ -225,6 +225,8 @@ public function getNocacheJs(): string
     for (const meta of document.querySelectorAll('meta[content=$csrfPlaceholder]')) {
         meta.content = data.csrf;
     }
+    
+    document.dispatchEvent(new CustomEvent('statamic::nocache-replaced'));
```

Comment:
Could you change this to `statamic:nocache.replaced`?

I tried to find some conventions / best practices and this was more inline with what I found.

```suggestion
    document.dispatchEvent(new CustomEvent('statamic:nocache.replaced'));
```